course-content/31_exploring_data_structures.qmd

```diff
@@ -435,14 +475,14 @@ You would like to summarize life expectancy by year for each continent so you ca
 Using what we learned about how to select rows, we should limit the dataframe to rows where the year is greater than or equal to 1997. Next we should create a multicolumn call to the `.groupby()` function. Finally we should select some aggregations such as mean, min, and max among others could make sense here.
 
 ```{python}
-df_current = df[df['year'] >= 1997]
+df_current = country_metrics[country_metrics['year'] >= 1997]
```

Comment:
For this challenge, I would make it that they need to summarize both life expectancy AND population for a variety of statistics. Getting stats for 2 columns isn't covered in the above text, but is straightforward enough based on what they've seen that they should be able to do it here. And we can walk through the example if enough people aren't getting it.

Reply:
Good idea. Restructured the challenge thusly